bin/xud

```diff
@@ -14,6 +14,11 @@ const { argv } = require('yargs')
       type: 'string',
       alias: 'x',
     },
+    loglevel: {
```

Comment:
Our convention for other args is snake_case with the underscore so let's stick to that, although I like e `-l` alias.

Reply:
Yeah you're right actually - I was confusing the `xucli` params which use snake_case with the `xud` and config args which use camel case. I'd be fine with switching xucli over to camel case too actually for the sake of consistency? That can definitely be a separate PR tho. I think I went with snake case earlier because the rpc fields in the proto definition use it, but that actually shouldn't matter to us.